### ISSUE TITLE
Weird BadSignature exception

### ISSUE THREAD

**Issue by tuxxy** [nucypher/nucypher#989]
```May 18 18:05:57 isogeny nucypher[8173]:   File "/usr/local/lib/python3.7/dist-packages/twisted/python/threadpool.py", line 250, in inContext
May 18 18:05:57 isogeny nucypher[8173]:     result = inContext.theWork()
May 18 18:05:57 isogeny nucypher[8173]:   File "/usr/local/lib/python3.7/dist-packages/twisted/python/threadpool.py", line 266, in <lambda>
May 18 18:05:57 isogeny nucypher[8173]:     inContext.theWork = lambda: context.call(ctx, func, *args, **kw)
May 18 18:05:57 isogeny nucypher[8173]:   File "/usr/local/lib/python3.7/dist-packages/twisted/python/context.py", line 122, in callWithContext
May 18 18:05:57 isogeny nucypher[8173]:     return self.currentContext().callWithContext(ctx, func, *args, **kw)
May 18 18:05:57 isogeny nucypher[8173]:   File "/usr/local/lib/python3.7/dist-packages/twisted/python/context.py", line 85, in callWithContext
May 18 18:05:57 isogeny nucypher[8173]:     return func(*args,**kw)
May 18 18:05:57 isogeny nucypher[8173]:   File "/home/tux/nucypher/nucypher/network/server.py", line 174, in learn_about_announced_nodes
May 18 18:05:57 isogeny nucypher[8173]:     certificate_filepath=certificate_filepath)
May 18 18:05:57 isogeny nucypher[8173]:   File "/home/tux/nucypher/nucypher/network/nodes.py", line 966, in verify_node
May 18 18:05:57 isogeny nucypher[8173]:     self.validate_metadata(accept_federated_only)  # This is both the stamp and interface check.
May 18 18:05:57 isogeny nucypher[8173]:   File "/home/tux/nucypher/nucypher/network/nodes.py", line 944, in validate_metadata
May 18 18:05:57 isogeny nucypher[8173]:     self.stamp_is_valid()
May 18 18:05:57 isogeny nucypher[8173]:   File "/home/tux/nucypher/nucypher/network/nodes.py", line 923, in stamp_is_valid
May 18 18:05:57 isogeny nucypher[8173]:     if self._stamp_has_valid_wallet_signature():
May 18 18:05:57 isogeny nucypher[8173]:   File "/home/tux/nucypher/nucypher/network/nodes.py", line 913, in _stamp_has_valid_wallet_signature
May 18 18:05:57 isogeny nucypher[8173]:     signature = EthSignature(signature_bytes)
May 18 18:05:57 isogeny nucypher[8173]:   File "/usr/local/lib/python3.7/dist-packages/eth_keys/datatypes.py", line 265, in __init__
May 18 18:05:57 isogeny nucypher[8173]:     raise BadSignature(str(err))
May 18 18:05:57 isogeny nucypher[8173]: eth_keys.exceptions.BadSignature: Value 27 is not less than or equal to 1
```

**Comment by jMyles** [comment]
@tuxxy: Can you post the value of `signature_bytes`?

**Comment by KPrasch** [comment]
Fixed in #951
